src/GUI.cpp

```diff
@@ -61,8 +61,20 @@ static void Display_DetailsWindow(HealWindowContext& pContext, DetailsWindowStat
 	ImGui::PushStyleColor(ImGuiCol_ChildBg, bgColor);
 	snprintf(buffer, sizeof(buffer), "##HEALDETAILS.TOTALS.%i.%llu", static_cast<int>(pDataSource), pState.Id);
 	ImGui::BeginChild(buffer, ImVec2(ImGui::GetWindowContentRegionWidth() * 0.35f, 0));
-	ImGui::Text("total healing");
-	ImGuiEx::TextRightAlignedSameLine("%llu", pState.Healing);
+
+	uint64_t adjustedHealing = (pState.Healing - pState.BarrierGeneration);
+
+	if (adjustedHealing > 0)
+	{
+		ImGui::Text("total healing");
+		ImGuiEx::TextRightAlignedSameLine("%llu", adjustedHealing);
+	}
+
+	if (pState.BarrierGeneration > 0)
+	{
+		ImGui::Text("total barrier gen");
+		ImGuiEx::TextRightAlignedSameLine("%llu", pState.BarrierGeneration);
+	}
```

Comment:
Maybe let's try to make sure at least one of them is always displayed? Something like
```
	if (adjustedHealing > 0 || pState.BarrierGeneration == 0)
	{
		ImGui::Text("total healing");
		ImGuiEx::TextRightAlignedSameLine("%llu", adjustedHealing);
	}

	if (pState.BarrierGeneration > 0)
	{
		ImGui::Text("total barrier gen");
		ImGuiEx::TextRightAlignedSameLine("%llu", pState.BarrierGeneration);
	}
```

Reply:
Changed to show both total and per second if there is no data to show:
![image](https://github.com/user-attachments/assets/a166051a-2f83-40ab-989a-5d2144e3b0b5)